menu/cakes/cakeA.html

```diff
@@ -0,0 +1,409 @@
+  <title>mandy b. bakes</title>
+  <meta property="og:title" content="Mandy B Bakes" />
+  <!-- <meta property="og:image" content="images/bg.jpg" /> -->
+  <meta property="og:description" content="Amanda B's Online Bakery" />
+  <meta property="og:url" content="https://yu-huanwu.github.io/mandybbakes/" />
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+  <link rel="stylesheet" href="../../assets/css/main.css" />
+  <link rel="icon" type="image/x-icon" href="../../images/favicon.ico" />
+</head>
+
+<body class="is-preload">
+
+  <!-- Wrapper -->
+  <div id="wrapper">
+
+    <!-- Main -->
+    <div id="main">
+      <div class="inner">
+
+        <!-- Header -->
+        <header id="header">
+          <a href="../../index.html" class="logo"><strong>Mandy B. Bakes</strong></a>
+          <ul class="icons">
+            <!-- <li><a href="#" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li> -->
+            <!-- <li><a href="#" class="icon brands fa-facebook-f"><span class="label">Facebook</span></a></li>
+										<li><a href="#" class="icon brands fa-snapchat-ghost"><span class="label">Snapchat</span></a></li> -->
+            <li><a href="https://instagram.com/mandyb.bakes" class="icon brands fa-instagram"><span
+                  class="label">Instagram</span></a></li>
+            <!-- <li><a href="#" class="icon brands fa-medium-m"><span class="label">Medium</span></a></li> -->
+          </ul>
+        </header>
+
+        <!-- Banner -->
+        <section id="banner">
+          <span class="image image-left object">
+            <img src="../../images/cakes/cake_pic03.jpg" alt="Amanda next to two-tiered wedding cake" />
+          </span>
+          <div class="content content-right">
+            <header>
+              <h1>Cake A</h1>
+              <!-- #TODO - ENTER PRICE BELOW -->
+              <!-- <p>$ X.XX</p> -->
+            </header>
+            <h2>Description:</h2>
+            <p>Aenean ornare velit lacus, ac varius enim ullamcorper eu. Proin aliquam facilisis ante interdum congue.
+              Integer mollis, nisl amet convallis, porttitor magna ullamcorper, amet egestas mauris. Ut magna finibus
+              nisi nec lacinia. Nam maximus erat id euismod egestas. Pellentesque sapien ac quam. Lorem ipsum dolor sit
+              nullam.</p>
+            <div>
+              <h3>Available Sizes:</h3>
+              <ul>
+                <li>4-inch (8 servings)</li>
+                <li>6-inch (12 servings)</li>
+                <li>8-inch (24 servings)</li>
+                <li>10-inch (34 servings)</li>
+              </ul>
+            </div>
+            <form class="orderForm" action="https://formspree.io/f/xvoepdpy" method="POST">
+            
+              <div>
+                <h3>Select Your Options:</h3>
+                
+                <!-- Pass Cake Name to Form -->
+                <label class="hidden">
```

Comment:
does this need to be hidden?

Reply:
actually i get what you are doing here now. Disregard my question, it's a good idea to keep this hidden while retaining the value of the specific cake name so that the owner will know what is being ordered.

Reply:
thanks, I renamed the "name" attribute to "itemName" to generalize it. Thinking maybe it will make it easier for Amanda to review the submission emails later (always look for "itemName" rather than X number of [product]Name fields).